articles/mfa/guides/configure-SMS-twilio.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Learn how to configure SMS with Twilio for MFA.
+topics:
+  - mfa
+  - twilio
+contentType:
+  - how-to
+useCase:
+  - customize-mfa
+---
+# Configure SMS with Twilio for MFA
+
+## SMS notifications
+
+Using SMS as an authentication factor means that the end user is sent a code via SMS when attempting to authenticate with your application. They will have to enter this code to complete the transaction, implying that in addition to knowing their login information, they also have possession of the device.
+
+::: warning
+When initially setting up SMS, you have up to 100 SMS to be used for testing. This limit can be removed by setting up a Twilio account. To prevent malicious login attempts, your users will always be limited to up to 10 SMS/Hour (replenishing one message an hour, up to 10).
+
+For testing Twilio, you should use the **Live Credentials** not the **Test Credentials** as the test credentials aren't meant to send real SMS messages.
+:::
+
+## End user experience
+
+![SMS End User 1](/media/articles/multifactor-authentication/mfa-sms1.png)
+
+After signing up, and entering a country code and phone number, the user will receive a six digit code to their device. They need to enter this code into the box in order to authenticate.
+
+![SMS End User 2](/media/articles/multifactor-authentication/mfa-sms2.png)
+
+Your users must have a device capable of receiving SMS messages to use this option. If users can't receive SMS messages, they will not be able to sign-up with this factor. If they already enrolled with SMS and don't have the device, they will need to use their recovery code to complete the MFA flow.
```

Comment:
```suggestion
Your users must have a device capable of receiving SMS messages to use this option. If users can't receive SMS messages, they will not be able to authenticate with this factor. If they already enrolled with SMS and don't have such a device, they will need to use their recovery code to complete the MFA flow.
```

Reply:
Adding this manually. I renamed the file based on Andres' suggestion.